src/components/UI/LazyImport/index.js

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 /**
  * After the lazy imported component loads, delay the promise to increase smoothness.
  */
-const LazyImport = props => {
+const LazyImport = (props) => {
```

Comment:
This is an eslint rule:

`Unexpected parentheses around single function argument.eslint(arrow-parens)`

Reply:
always presenting parens make refactoring easier: when you need to add another arg you don't need to add parens (cause they are present already).